cluster/cluster-master.yml

```diff
@@ -13,7 +13,7 @@ spec:
     spec:
       containers:
       - name: couchbase-master
-        image: arungupta/couchbase
+        image: rsmitty/couchbase
```

Comment:
rsmitty/couchbase is just a docker build from this clusters/ directory. So it incorporates the changes to configure-node.sh